patches/0007-Remove-bundled-mozilla-extensions.patch

```diff
@@ -12,14 +12,14 @@ index 974c48c612..0d1e95fa22 100644
 +++ b/browser/extensions/moz.build
 @@ -5,9 +5,6 @@
  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
- 
+
  DIRS += [
 -    'doh-rollout',
 -    'formautofill',
 -    'screenshots',
```

Comment:
We could squash this change with patch 3 so all the changes to this file are done in a single commit.

Reply:
Removing a commit form patch list will require to re-name all patches so I would like to do it in separate PR.